.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,22 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
```

Comment:
Is this added the right way ... Shouldn't it just flow from the sync service?

Reply:
The sync service is broken right now due to SSO, so adding it manually